source/network/network.go

```diff
@@ -19,19 +19,20 @@ package network
 import (
 	"bytes"
 	"fmt"
+	"github.com/golang/glog"
 	"io/ioutil"
 	"net"
 	"strconv"
 	"strings"
-	"github.com/golang/glog"
 )
 
 // Source implements FeatureSource.
 type Source struct{}
 
+// Name returns a friendly name for this source.
```

Comment:
Maybe change this description to `Name returns a identifier string for this feature source`?